crates/rpc-types-eth/src/transaction/receipt.rs

```diff
@@ -67,11 +67,6 @@ pub struct TransactionReceipt<T = ReceiptEnvelope<Log>> {
     pub to: Option<Address>,
     /// Contract address created, or None if not a deployment.
     pub contract_address: Option<Address>,
-    /// The post-transaction stateroot (pre Byzantium)
-    ///
-    /// EIP98 makes this optional field, if it's missing then skip serializing it
-    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
-    pub state_root: Option<B256>,
```

Comment:
yep we def need to remove this here because this is handled by the receipt
<img width="970" alt="image" src="https://github.com/user-attachments/assets/eb5ce343-6962-4a0b-89a9-05afc8a7482d">